dali/pipeline/executor/executor.cc

```diff
@@ -65,14 +65,23 @@ void Executor<WorkspacePolicy, QueuePolicy>::SyncDevice() {
 template <typename WorkspacePolicy, typename QueuePolicy>
 void Executor<WorkspacePolicy, QueuePolicy>::RunCPUImpl() {
   PreRun();
-
+  static std::string placement_error =
```

Comment:
const?

Reply:
We don't need an instance of std string here. How about a compile-time constant?
```suggestion
  const char placement_error[] =
```